spec/index.html

```diff
@@ -908,6 +949,221 @@ <h3>Converting a <a>YAML scalar</a></h3>
     </section>
   </section>
 
+  <section id="convert-to">
+    <h2>Converstion to YAML</h2>
+
+    <p>The conversion process from the <a>internal representation</a>
+      involves turning that representation back into a YAML
+      <a data-cite="YAML#representation-graph"></a>
+      and relies on the descripition of
+      "Serializing the Representation Graph" from the
+      <a data-cite="YAML#dump">3.1.1 Dump</a> section of [[YAML]]
+      for the final transformation to serialized YAML.
+    </p>
+
+    <p>
+      As the <a>internal representation</a> is rooted by either
+      an <a>array</a> or a <a>map</a>,
+      the process of transforming the <a>internal representation</a>
+      to YAML begins by preparing an empty <a>representation graph</a>
+      which will rooted with either a
+      <a>YAML mapping</a> or <a>YAML sequence</a>.
+      Although outside of the scope of this specification,
+      processors MAY use
+      <a>YAML directives</a> including <a>TAG directives</a>, and
+      <a data-cite="YAML#document-markers">Document markers</a>
+      as appropriate for best serialization results.
+      Specifically, if the `extendedYAML` API flag is `true`,
+      the document SHOULD use the `%YAML` directive with
+      version set to at least `1.2`
+      and a `%TAG` directive appropriate for each
+      `RDF literal` having a datatype
+      (or `i18n` for <a>language-tagged strings</a>).
```

Comment:
It's unclear how and to what this parenthetical applies.  Probably best if the text can be rephrased to not require a parenthetical afterthought at all, but states everything straight out. (The `<h4>Converting...` sections which follow immediately below should also include `language-tagged strings` somewhere, corresponding to the rephrase above.)

Reply:
See 75355aa89e5d859e5c5983027341d048840db896.